test/jdk/TEST.groups

```diff
@@ -385,16 +385,33 @@ jdk_editpad = \
      jdk/editpad
 
 jdk_desktop = \
-    :jdk_awt \
-    :jdk_2d \
-    :jdk_beans \
+    :jdk_desktop_part1 \
+    :jdk_desktop_part2 \
+    :jdk_desktop_part3
+
+jdk_desktop_part1 = \
+    :jdk_client_sanity \
     :jdk_swing \
+    :jdk_2d \
     :jdk_sound \
     :jdk_imageio \
-    :jdk_accessibility \
+    :jdk_editpad \
     :jfc_demo \
-    :jdk_client_sanity \
-    :jdk_editpad
+    :jdk_accessibility \
+    :jdk_beans
+
+jdk_desktop_part2 = \
+    :jdk_awt \
+    -java/awt/Component \
+    -java/awt/Modal \
+    -java/awt/datatransfer \
+    -java/awt/Window
+
+jdk_desktop_part3 = \
+    java/awt/Component \
+    java/awt/Modal \
+    java/awt/datatransfer \
+    java/awt/Window
```

Comment:
I am not sure I understand why we need to run java/awt/Component, java/awt/Modal, java/awt/datatransfer, java/awt/Window in part_2 and part_3...Will it not be duplication of run of tests (as jdk_desktop is supposed to run all part1,2,3)?

Reply:
We aren't. the "-" means it is excluded.

Reply:
ok...Thanks for the clarification..
Probably we could do the same for javax/swing instead of running in its entirety..

Reply:
There is no point. The entire jdk_swing group runs in far less time than that.